app/grandchallenge/algorithms/static/algorithms/js/display_result_text.mjs

```diff
@@ -12,3 +12,8 @@ $('#resultInfoModal').on('show.bs.modal', function (event) {
     modal.find('.modal-title').text(button.data('title'));
     modal.find('#resultDetailLink').attr("href", `${button.data("pk")}/`);
 })
+
+$('[href="#v-pills-logs"]').on('shown.bs.tab', function (event) {
```

Comment:
I think this works, but just curious: why not the id of the target element (i.e. `v-pills-logs-tab`)?

Reply:
I'll replace that! I had a good reason, but on further digging it's not needed in this version of bootstrap. Just tested and it seems fine to just swap to the element